tools/cpp_build/build_common.sh

```diff
@@ -31,14 +32,15 @@ BUILD_TYPE=${BUILD_TYPE:-Debug}
 # Try to build with as many threads as we have cores, default to 4 if the
 # command fails.
 set +e
+JOBS=4
 if [[ "$(uname)" -eq "Linux" ]]; then
   # https://stackoverflow.com/questions/6481005/how-to-obtain-the-number-of-cpus-cores-in-linux-from-the-command-line
   JOBS="$(grep -c '^processor' /proc/cpuinfo)"
-else # if [[ "$(uname)" -eq "Darwin"]]
+elseif [[ "$(uname)" -eq "Darwin"]]
   # https://stackoverflow.com/questions/1715580/how-to-discover-number-of-logical-cores-on-mac-os-x
   JOBS="$(sysctl -n hw.ncpu)"
 fi
 set -e
-if [[ $? -ne 0 ]]; then
-  JOBS=4
-fi
+#if [[ $? -ne 0 ]]; then
+#  JOBS=4
+#fi
```

Comment:
Why did this change?

Reply:
I fixed this #6420, I'm removing this change.